BlueZ 5.46 error: No such field: NotifyAcquired in obj, No such field: WriteAcquired in obj

It looks like there is an issue trying to use this library with the latest version of BlueZ. I used your script to upgrade to version 5.46 so that I could use the new [acquire-write and acquire-notify](https://unix.stackexchange.com/questions/392103) features. But now when I try using `api.GetDeviceByAddress(deviceID)` I get the following error messages:

```
root@GBRPI01:~/go/src/gitlab.com/gbrayut/rpi# go run cmd/ble-test/main.go
Starting BLE test app
hciconfig status: &linux.HCIConfigResult{Enabled:true, Address:"B8:27:EB:CF:0F:5B", Type:"Primary", Bus:"UART"}
panic: No such field: NotifyAcquired in obj

goroutine 1 [running]:
gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/bluez/profile.NewGattCharacteristic1(0x11aea180, 0x3a, 0x11aea180)
        /home/gbray/go/src/gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/bluez/profile/GattCharacteristic1.go:27 +0x1a4
gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/api.(*Device).GetCharByUUID(0x11ab67c0, 0x11afaf90, 0x24, 0x0, 0x0, 0x70)
        /home/gbray/go/src/gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/api/Device.go:256 +0x328
main.main()
        /home/gbray/go/src/gitlab.com/gbrayut/rpi/cmd/ble-test/main.go:59 +0x1a4
exit status 2


root@GBRPI01:~/go/src/gitlab.com/gbrayut/rpi# go run cmd/ble-test/main.go
Starting BLE test app
hciconfig status: &linux.HCIConfigResult{Enabled:true, Address:"B8:27:EB:CF:0F:5B", Type:"Primary", Bus:"UART"}
panic: No such field: WriteAcquired in obj

goroutine 1 [running]:
gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/bluez/profile.NewGattCharacteristic1(0x1161c380, 0x3a, 0x1161c380)
        /home/gbray/go/src/gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/bluez/profile/GattCharacteristic1.go:27 +0x1a4
gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/api.(*Device).GetCharByUUID(0x11655360, 0x11616e40, 0x24, 0x0, 0x0, 0x70)
        /home/gbray/go/src/gitlab.com/gbrayut/rpi/vendor/github.com/muka/go-bluetooth/api/Device.go:256 +0x328
main.main()
        /home/gbray/go/src/gitlab.com/gbrayut/rpi/cmd/ble-test/main.go:59 +0x1a4
exit status 2
```

These seem to be from the new features in 5.46. I was able to add the two missing properties to GattCharacteristic1Properties in my vendor\github.com\muka\go-bluetooth\bluez\profile\GattCharacteristic1.go file and it started working again 

```
// GattCharacteristic1Properties exposed properties for GattCharacteristic1
type GattCharacteristic1Properties struct {
	Value          []byte
	Notifying      bool
	NotifyAcquired bool
	WriteAcquired  bool
	Service        dbus.ObjectPath
	UUID           string
	Flags          []string
	Descriptors    []dbus.ObjectPath
}
```

Not sure if there is a way to make it backwards compatible or not, but just thought I would give you a heads up.

--- a/bluez/profile/GattCharacteristic1.go
+++ b/bluez/profile/GattCharacteristic1.go
@@ -41,6 +41,8 @@ type GattCharacteristic1 struct {
 type GattCharacteristic1Properties struct {
 	Value       []byte
 	Notifying   bool
+	NotifyAcquired bool
+	WriteAcquired  bool
 	Service     dbus.ObjectPath
 	UUID        string
 	Flags       []string

